package/MDAnalysis/analysis/align.py

```diff
@@ -1004,22 +1005,46 @@ def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
 
     See Also
     --------
-    BioPython documentation for `pairwise2`_. Alternatively, use
+    BioPython documentation for `PairwiseAligner`_. Alternatively, use
     :func:`fasta2select` with :program:`clustalw2` and the option
     ``is_aligned=False``.
 
-    .. _`pairwise2`: http://biopython.org/DIST/docs/api/Bio.pairwise2-module.html
+
+    .. _`PairwiseAligner`:
+       https://biopython.org/docs/latest/api/Bio.Align.html#Bio.Align.PairwiseAligner
+
 
     .. versionadded:: 0.10.0
 
-    """
-    aln = Bio.pairwise2.align.globalms(
-        reference.residues.sequence(format="string"),
-        mobile.residues.sequence(format="string"),
-        match_score, mismatch_penalty, gap_penalty, gapextension_penalty)
-    # choose top alignment
-    return aln[0]
+    .. versionchanged:: 2.4.0
+       Replace use of deprecated :func:`Bio.pairwise2.align.globalms` with
+       :class:`Bio.Align.PairwiseAligner`.
 
+    """
+    aligner = Bio.Align.PairwiseAligner(
+        mode="global",
+        match_score=match_score,
+        mismatch_score=mismatch_penalty,
+        open_gap_score=gap_penalty,
+        extend_gap_score=gapextension_penalty)
+    aln = aligner.align(reference.residues.sequence(format="Seq"),
+                        mobile.residues.sequence(format="Seq"))
+    # choose top alignment with highest score
+    topalignment = aln[0]
+
+    # reconstruct the results tuple that used to be of type Bio.pairwise2.Alignment
+    AlignmentTuple = collections.namedtuple(
+        "Alignment",
+        ["seqA", "seqB", "score", "start", "end"])
+    # extract sequences (there's no obvious way to get the character
+    # representation with gaps by other means from the new
+    # Bio.Align.PairwiseAlignment instance)
+    seqA, _, seqB, _ = topalignment.format().split("\n")
```

Comment:
This only worked until Bio 1.79. Then they changed the format.

I knew that parsing str was iffy but luckily our tests showed me right away _how_ iffy that was.

I am now trying to find another way to get our old output.

Reply:
Turns out, it's trivial
```python
seqA = topalignment[0]
seqB = topalignment[1]
```
... testing now.